api/clients/eigenda_client.go

```diff
@@ -205,15 +232,11 @@ func (m *EigenDAClient) putBlob(ctx context.Context, rawData []byte, resultChan
 	}
 	// disperse blob
 	// TODO: would be nice to add a trace-id key to the context, to be able to follow requests from batcher->proxy->eigenda
-	blobStatus, requestID, err := m.Client.DisperseBlobAuthenticated(ctx, data, customQuorumNumbers)
+	_, requestID, err := m.Client.DisperseBlobAuthenticated(ctx, data, customQuorumNumbers)
```

Comment:
since err is wrapped, the proxy client should handle this error

Reply:
Not sure what you are saying here. Are you saying the wrapping here will break the eigenda-proxy logic? It won't, this error will be handled fine, the wrapping doesn't change anything, because grpc's `status.FromError()` function will unwrap until it find a grpc error.

Reply:
just a place holder, so that comment from the other PR can reference